### PR TITLE
Spherical bessel function for 'z=0' in jn(nu, z)

### LINKED ISSUE #11776: jn(0, 0) giving nan
Reported at https://stackoverflow.com/questions/40232986/spherical-bessel-transformation-using-sympy

`jn(0, 0)` remains unevaluated, but `jn(0, 0).rewrite(besselj)` and `jn(0, 0).evalf()` both give `nan`. The answer should be 1.

### PR DESCRIPTION
**Pull request by darknight009**
added checks for jn(n,0) for n>=0.

<!-- If this pull request fixes an issue please indicate which issue by typing "Fixes #NNNN" below.>

Fixes #11776

### CODE REVIEW

File: sympy/functions/special/bessel.py [hunk 1]
```diff
@@ -668,7 +668,17 @@ class jn(SphericalBesselBase):
     .. [1] http://dlmf.nist.gov/10.47
 
     """
+    _a = S.One
+    _b = S.One
```
Comment (asmeurer) [inline sympy/functions/special/bessel.py]: Why did you add these lines?

Comment (dranknight09) [inline sympy/functions/special/bessel.py]: I must have left that stray code. That needs to go ;)

**Comment by asmeurer** [comment]
Tests are failing. You forgot to add `@classmethod` above the `eval` definition.

**Comment by gxyd** [comment]
@dranknight09 can you please add test case for the PR? Let me know if you need any help with adding test cases.

**Comment by gxyd** [comment]
@jksuom please review.

**Comment by gxyd** [comment]
There was some discussion on https://github.com/sympy/sympy/pull/12505 regarding 'supporting Spherical Bessel functions' for complex orders.

**Comment by jksuom** [comment]
It would probably suffice to explicitly evaluate only functions of (non-negative) integer order as defined in the reference.

**Comment by czgdp1807** [comment]
@darknight009 Would you like to continue this PR?
@asmeurer @jksuom @gxyd Your opinions?

**Comment by darknight009** [comment]
@czgdp1807  I will not be continuing.
